src/Admin/OnBoarding/StepAbstract.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+namespace WP_SMS\Admin\OnBoarding;
+
+use WP_SMS\Components\View;
+use WP_SMS\Utils\Request;
+use WP_SMS\Utils\Validator;
+
+abstract class StepAbstract
+{
+    public $title;
+    private $skippable = true;
+    private $wizard;
+
+    protected $data = [];
+    private $fields = [];
+    private $errors = [];
+
+    public function __construct(WizardManager $wizard)
```

Comment:
Let's make sure the Type Hints is supported in PHP v5.6 (seems yes → https://3v4l.org/cEin7)